Runtime/EcsactRuntime.cs

```diff
@@ -624,70 +624,17 @@ public struct AsyncEventsCollector {
 		public IntPtr actionCommittedCallbackUserData;
 	}
 
-	private static EcsactRuntime? defaultInstance;
-
 	static EcsactRuntime() {
 #if UNITY_EDITOR
-		EditorApplication.playModeStateChanged += state => {
-			if(state == PlayModeStateChange.ExitingPlayMode) {
-				EditorApplication.delayCall += () => {
-					SetDefault(null);
-				};
-			}
-		};
-#endif
-	}
-
-	public static EcsactRuntime GetOrLoadDefault() {
-		if(!UnityEngine.Application.isPlaying) {
-			throw new Exception(
-				"EcsactRuntime.GetOrLoadDefault() may only be used during play mode"
-			);
-		}
-
-		if(defaultInstance == null) {
-			var settings = EcsactRuntimeSettings.Get();
-			defaultInstance = Load(settings.runtimeLibraryPaths);
-
-			if(defaultInstance != null) {
-				Ecsact.Internal.EcsactRuntimeDefaults.Setup(defaultInstance, settings);
-			}
-		}
-
-		if(defaultInstance == null) {
-#if UNITY_EDITOR
-			UnityEditor.EditorApplication.isPlaying = false;
-			var okQuit = UnityEditor.EditorUtility.DisplayDialog(
-				title: "Failed to load default ecsact runtime",
-				message: "Please check your ecsact runtime settings",
-				ok: "Ok Quit",
-				cancel: "Continue Anyways"
-			);
-
-			if(okQuit) {
-				UnityEditor.EditorApplication.isPlaying = false;
-			}
-			UnityEngine.Application.Quit(1);
-#else
-			UnityEngine.Debug.LogError("Failed to load default ecsact runtime");
-			UnityEngine.Application.Quit(1);
+		// NOTE(Kelwan): Do I need this somehow?
+		// EditorApplication.playModeStateChanged += state => {
+		// 	if(state == PlayModeStateChange.ExitingPlayMode) {
+		// 		EditorApplication.delayCall += () => {
+		// 			SetDefault(null);
+		// 		};
+		// 	}
+		// };
```

Comment:
Because unity runs in a single process all the state persists when exiting play mode. What this did before is made sure the default runtime was `null` after exiting play mode. That made it nice and clean when starting play mode again.

Probably settings all the `Defaults.*` to `null` when `playModeStateChanged` is fired with `ExitingPlayMode` would be good.